fl-server/src/handlers.rs

```diff
@@ -268,12 +334,13 @@ pub async fn get_flist_state_handler(
 pub async fn list_flists_handler(State(state): State<Arc<config::AppState>>) -> impl IntoResponse {
```

Comment:
does this list all flists or per user? what if we have too many of them? should you consider pagination ? and or filtering as well.

Does it make sense to use some small db (say sqlite) to track the flists instead of scanning the entire directory of all flists everytime ?

The db can be recreated from fs when/if needed but it will be way more efficient to hit the db for queries like what flists we have, what users, what flists for that user, etc.. than scanning the fs

Reply:
https://github.com/threefoldtech/rfs/issues/77